.gitignore

```diff
@@ -1,5 +1,6 @@
 .idea
 *.iml
+*.swp
```

Comment:
It is a better practice to use a global .gitignore file for that: https://help.github.com/articles/ignoring-files/#create-a-global-gitignore